takwimu/takwimu_ui/src/components/AnalysisContent/Actions/DownloadPDF.js

```diff
@@ -235,8 +235,11 @@ function DownloadPDF({ classes, title, topic, data, takwimu }) {
 
   return (
     <ButtonBase
-      disableRipple
-      disableTouchRipple
+      ga-on="click"
+      ga-event-category="Analysis"
+      ga-event-action="Download"
+      ga-event-label={`${takwimu.country.name}: ${title}`}
+      ga-event-value={top ? 'Top' : 'Bottom'}
```

Comment:
`eventValue` needs to be a non-negative integer @karimkawambwa: https://developers.google.com/analytics/devguides/collection/analyticsjs/field-reference#eventValue